community/cryptsetup/depends

```diff
@@ -0,0 +1,3 @@
+json-c make
```

Comment:
`make` means the dependencies required at compile time and not at runtime

`openssl`, `json-c` and `lvm2` are probably required at runtime so they shouldn't be marked as make deps

Reply:
I configured cryptsetup to compile static binaries, so that shouldn't be the case. That way you don't need to include its dependencies when using in an initramfs. If that's a problem, I'll change it.